docs/content/installation/installation-with-manifests.md

```diff
@@ -101,6 +101,7 @@ Create a custom resource definition for `APPolicy`, `APLogConf` and `APUserSig`:
    $ kubectl apply -f common/crds/appprotect.f5.com_appolicies.yaml
    $ kubectl apply -f common/crds/appprotect.f5.com_apusersigs.yaml
    ```
+> Skip steps above if you are using App Protect WAF module with policy bundles.
```

Comment:
I think it would be better for this note to be mentioned before these steps instead of after. It's possible a user will apply these CRDs and then read this note after the fact.

So in this case, we can move this to L96 and change it to this:

```suggestion
> **Note** This step can be skipped if you are using App Protect WAF module with policy bundles.
```